src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -441,6 +441,18 @@ end</script>
 				<packageName></packageName>
 				<regex>^map translate (\w+) (\w+) (\w+)$</regex>
 			</Alias>
+			<Alias isActive="yes" isFolder="no">
+				<name>Clear All Map Data Alias</name>
+				<script>for id, name in pairs(getAreaTableSwap()) do
```

Comment:
since we added the function clearAllMapData to Other.lua, we should just call it here instead of repeating the code.

Reply:
hmm, Actually, n/m, if the mapper gets pushed first then the function won't exist yet.

Reply:
> since we added the function clearAllMapData to Other.lua, we should just call it here instead of repeating the code.

en, or delete this alias?

Reply:
The alias is fine as is, since the mapper updates independently of Mudlet itself we can't count on the function being there until after the next release.  

You're good to go =)

Reply:
ok, We can update this alias after the function takes effect.